components/sidebar/PatternClassAccordionPattern.tsx

```diff
@@ -8,12 +10,21 @@ type Props = {
 const PatternClassAccordionPattern = ({ pattern }: Props) => {
   const { patternNames } = useSelection()
   return (
-    <div className="flex justify-between" key={pattern.name}>
-      <div className="overflow-hidden text-ellipsis w-full whitespace-nowrap">
-        {pattern.name}
-      </div>
+    <div key={pattern.name} className={css.root}>
+      <label className={css.label} htmlFor={pattern._id}>
+        <div className={css.labelText}>
+        <Hotel size={24} className={css.icon} />
+        <p>{pattern.name}</p>
+        </div>
+        {patternNames.includes(pattern.name) ?
+          <RadioButtonChecked size={24} className={css.checkbox} /> :
+          <RadioButton size={24} className={css.checkbox} />
```

Comment:
these round radio-styled checkboxes make me very a11y weary :see_no_evil: but i don't think it needs to be a concern at this stage!

Reply:
Yeah I agree - it's almost certainly not up to spec but seemed ok for now also.

I think ideally we'd need to reference the input checked value directly instead of checking the same condition. The checkbox is still here and doing it's thing, just hidden.


https://user-images.githubusercontent.com/20502206/212736223-a768dbc9-7019-4001-8fa6-f6fba9fe5789.mov